src/utils2.js

```diff
@@ -0,0 +1,44 @@
+// helper functions
```

Comment:
This util file should be named more intelligentally. dom_util maybe?

Reply:
agree.